public/base-locales/de.json

```diff
@@ -439,11 +439,11 @@
   "everything_individually": "Individuelle Benachrichtigungen hinzufügen",
   "individually": "individuell",
   "errors_404": "Hoppla, diese Seite kann nicht gefunden werden!",
-  "errors_500": "Es ist ein Serverfehler aufgetreten, versuchen Sie es später erneut!",
+  "errors_500": "Es ist ein Serverfehler aufgetreten, bitte versuche es später erneut!",
   "local_username": "Nutzername",
   "local_password": "Passwort",
-  "local_error": "Es ist ein Fehler aufgetreten, bitte versuchen es später noch einmal!",
-  "no_filter_results": "Keine Ergebnisse gefunden\nVersuche, Deine Filter zu erweitern",
+  "local_error": "Es ist ein Fehler aufgetreten, bitte versuche es später noch einmal!",
+  "no_filter_results": "Keine Ergebnisse gefunden,\nversuche Deine Filter zu erweitern!",
```

Comment:
`deine Filter` :)

Reply:
Oh think im to oldschool .... salutations in letters will start with capital letters, in german. 
Maybe there are more of that! -> But this is changed!